skfda/exploratory/visualization/_parametric_plot.py

```diff
@@ -28,65 +40,84 @@ class ParametricPlot:
             has dim_codomain = 1.
+        chart: figure over with the graphs are plotted or axis over
+            where the graphs are plotted. If None and ax is also
+            None, the figure is initialized.
+        fig: figure over with the graphs are plotted in case ax is not
+            specified. If None and ax is also None, the figure is
+            initialized.
+        ax: axis where the graphs are plotted. If None, see param fig.
     """
 
     def __init__(
         self,
         fdata1: FData,
         fdata2: Optional[FData] = None,
+        chart: Union[Figure, Axes, None] = None,
+        *,
+        fig: Optional[Figure] = None,
+        axes: Optional[Axes] = None,
+        group: Optional[Sequence[K]] = None,
+        group_colors: Optional[Indexable[K, ColorLike]] = None,
+        group_names: Optional[Indexable[K, str]] = None,
+        legend: bool = False,
     ) -> None:
+        BasePlot.__init__(self)
         self.fdata1 = fdata1
         self.fdata2 = fdata2
 
+        if self.fdata2 is not None:
+            self.fd_final = self.fdata1.concatenate(
+                self.fdata2, as_coordinates=True,
+            )
+        else:
+            self.fd_final = self.fdata1
+
+        self.group = group
+        self.group_names = group_names
+        self.group_colors = group_colors
+        self.legend = legend
+
+        self._set_figure_and_axes(chart, fig, axes)
+
     def plot(
         self,
-        chart: Union[Figure, Axes, None] = None,
-        *,
-        fig: Optional[Figure] = None,
-        ax: Optional[Axes] = None,
-        **kwargs: Any,
     ) -> Figure:
         """
         Parametric Plot graph.
 
         Plot the functions as coordinates. If two functions are passed
         it will concatenate both as coordinates of a vector-valued FData.
-        Args:
-            chart: figure over with the graphs are plotted or axis over
-                where the graphs are plotted. If None and ax is also
-                None, the figure is initialized.
-            fig: figure over with the graphs are plotted in case ax is not
-                specified. If None and ax is also None, the figure is
-                initialized.
-            ax: axis where the graphs are plotted. If None, see param fig.
-            kwargs: optional arguments.
         Returns:
-            fig (figure object): figure object in which the ParametricPlot
+            fig: figure object in which the ParametricPlot
             graph will be plotted.
         """
-        fig, axes = _get_figure_and_axes(chart, fig, ax)
+        self.artists = np.zeros(self.n_samples(), dtype=Artist)
 
-        if self.fdata2 is not None:
-            self.fd_final = self.fdata1.concatenate(
-                self.fdata2, as_coordinates=True,
-            )
-        else:
-            self.fd_final = self.fdata1
+        sample_colors, patches = _get_color_info(
```

Comment:
Where is this used?

Reply:
i was gonna finish it now, yesterday i was finishing it before the meeting